ragna/deploy/_ui/modal_configuration.py

```diff
@@ -199,10 +228,7 @@ async def model_section(self):
         )
 
     @pn.depends("config", "config.assistant_name", "config.source_storage_name")
-    def advanced_config_ui(self):
-        if self.config is None:
-            return
-
+    def advanced_config(self, is_corpus=False):
```

Comment:
We now need to support two different configs - one for corpus, and one for file upload, as they have different features.